<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nama Barang</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVendor">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNamaBarang">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Harga Barang</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Vendor Barang</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHargaBarang">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditJumlahBarang">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Jumlah Barang</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableBarang">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>761</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nama Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vendor Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEditBayar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTambahTransaksi">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Total :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotal">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pembayaran</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kembalian :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelKembalian">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBayar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>330</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Bayar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
